tools/wptrunner/wptrunner/manifestupdate.py

```diff
@@ -931,7 +931,7 @@ def get_manifest(metadata_root, test_path, url_base, run_info_properties, update
     :param url_base: Base url for serving the tests in this manifest"""
     manifest_path = expected.expected_path(metadata_root, test_path)
     try:
-        with open(manifest_path) as f:
+        with open(manifest_path, "rb") as f:
```

Comment:
This implies that manifest files would be binary, which seems surprising to me. Aren't they essentially just `.ini` files - what data in them requires reading them as binary?

Reply:
Honestly not sure, but the parser asserts that it's binary.

https://github.com/web-platform-tests/wpt/blob/e2c33e25126008057a41966463f064cc3d08a580/tools/wptrunner/wptrunner/wptmanifest/parser.py#L91

The file must be opened as binary to pass this assertion.

Reply:
Well, I can't argue with that :). LGTM for now, this may be something we want to visit one day in the future.